### DISCUSSION TITLE
Как сделать печать таблицы с помощью linkToPrint

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#542] [Как это сделать?] [answered]
Есть задача вывести на печать двумерную таблицу с помощью linkToPrint. У таблицы должна быть шапка ( имя полей ) и строки с данными из таблицы ТОТУМ  отобранные с помощью selectList.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Это делается через вложенные шаблоны: https://ru.docs.totum.online/embedded-templates (нет там видоса к сожалению)
